lib/fh_cluster.js

```diff
@@ -33,7 +35,7 @@ var g_numWorkers;
  *   - startEventId: Event That Will Start The Worker
  *   - preferredWorkerId: Preferred Worker ID To Bind The Task To
  */
-module.exports = function fhCluster(workerFunc, optionalNumWorkers, optionalBackoffStrategy, boundWorkerTasks) {
+module.exports = function fhCluster(workerFunc, optionalNumWorkers, optionalBackoffStrategy, boundWorkerTasks, gracefullShutdownFn) {
```

Comment:
Not important: `gracefull` -> `graceful`

Possibly more importantly, there's a doc-style comment above this function - I'd suggest adding the new parameter to that.